packages/tabs/src/styled/StyledTab.ts

```diff
@@ -5,55 +5,102 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import classNames from 'classnames';
-import TabStyles from '@zendeskgarden/css-tabs';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import styled, { DefaultTheme, css, ThemeProps } from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'tabs.tab';
 
 interface IStyledTabProps {
-  isHovered?: boolean;
-  isFocused?: boolean;
-  isActive?: boolean;
   disabled?: boolean;
   isSelected?: boolean;
 }
 
+/**
+ * A high specificity is needed to apply the border-color in vertical orientations
+ */
+const colorStyles = ({ theme, isSelected }: IStyledTabProps & ThemeProps<DefaultTheme>) => css`
+  color: ${isSelected ? getColor('primaryHue', 600, theme) : 'inherit'};
+
+  &&& {
```

Comment:
or better, just calculate `border-color` the same way that you do with `color` above:

```css
border-color: ${isSelected ? getColor(...) : 'transparent'};
```

Reply:
This still requires a specificity hack to beat the styling. Since we have to override the default border-color of `transparent` provided in `StyledTabs`.

This hack allows us to avoid using context to provide an `isVertical` prop which we spread to all the components so I think it's still worth it to include.